packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/components/RecentSubjects/RecentSubjectsCarousel.js

```diff
@@ -0,0 +1,39 @@
+import counterpart from 'counterpart'
+import { arrayOf, shape, string } from 'prop-types'
+import React from 'react'
+import { Carousel } from 'grommet'
+import SubjectThumbnail from './components/SubjectThumbnail'
+
+// TODO: Use the subject viewers from the classifier
+function RecentSubjectsCarousel (props) {
+  const { className, href, subjects } = props
+  const height = 500
+  const width = 700
+  return (
+    <Carousel
+      controls='arrows'
```

Comment:
`controls='arrows'` isn't supported in a public release of Grommet yet. See https://github.com/grommet/grommet/pull/3176